src/components/sbb-link/sbb-link.tsx

```diff
@@ -160,21 +176,22 @@ export class SbbLink implements LinkButtonProperties, ComponentInterface {
 
   public render(): JSX.Element {
     let TAG_NAME: string;
-    let attributeList: object = getLinkButtonBaseAttributeList(
-      this.idValue,
-      this._getClassString(),
-      this
-    );
+    let attributeList: object = getLinkButtonBaseAttributeList(this);
```

Comment:
Minor performance nitpick; Are we not iterating the accessibility properties twice with the calls to `getLinkButtonBaseAttributeList` and `getLinkButtonAttributeList`?

Reply:
Yes it's called twice. I don't know which solution would be the best to avoid. generally I like if it is possible to just call `getLinkButtonAttributeList` and you have all properties in it (also the base props). I thought the performance impact would be minimal as it is (also removed Object assign now in accessibility properties). Otherwise we could pass the base attributes into `getLinkButtonAttributeList` and make a switch there?

Reply:
Forget my answer, is much easier to solve on callers side :D